frontend/src/metabase/dashboard/selectors/selectors.js

```diff
@@ -77,6 +77,9 @@ export const getIsShowDashboardInfoSidebar = createSelector(
   sidebar => sidebar.name === SIDEBAR_NAME.info,
 );
 
+/**
+ * @type {(state: import("metabase-types/store").State) => import("metabase-types/api").Dashboard}
```

Comment:
This ensures TypeScript files using `getDashboard` will get the correct type without having to convert this whole file to TypeScript which is not trivial